app/components/channel_list/categories/body/channel/channel_list_item.tsx

```diff
@@ -126,7 +140,7 @@ const ChannelListItem = ({channel, isActive, currentUserId, isMuted, myChannel,
         <Animated.View style={animatedStyle}>
             <TouchableOpacity onPress={switchChannels}>
                 <View
-                    style={styles.container}
+                    style={[styles.container, isActive && styles.activeItem]}
```

Comment:
This may re-render more than needed. Nevertheless, can be measured and addressed in the future.

Reply:
Why would it? I'm definitely not a fan of memoizing styles without need.